src/locales/en/en.json

```diff
@@ -75,7 +76,14 @@
   "USER_BLOCK_SUUCCESSFULLY": "User Block Successfully",
   "USER_UNBLOCK_SUUCCESSFULLY":"User Unblock Successfully",
 
-  "COMMON_USER":"Users",
+  "USER_SEARCH_FIRST_NAME":"Search Firstname",
+  "USER_SEARCH_LAST_NAME":"Search Lastname",
+  "USER_SEARCH_ORGANIZATION":"Search Organization",
```

Comment:
Is this language constant used anywhere in PR?

Reply:
removed unused lang.constant